src/id/resources/profile/index.ts

```diff
@@ -319,15 +319,23 @@ export default class Profile extends IDResource {
     })
   }
 
-  // TODO Add Dates
-  // getDates () {
-  //   console.log('attributes_dates')
-  // }
+  getDates (): Promise<Array<ProfileDatesAttribute>> {
+    return this.client.call({
+      method: 'get',
+      endpoint: '/profiles/me/attributes/dates'
+    }).then((response) => {
+      return response.data
+    })
+  }
 
-  // TODO Add Dates by Attribute
-  // getDatesByAttribute () {
-  //   console.log('attributes_dates_datesAttribute')
-  // }
+  deleteDate (dateId: string): Promise<Array<ProfileDatesAttribute>> {
```

Comment:
Promise<ProfileDatesAttribute[]>
just to be consistent with the rest of the repo

Reply:
changed.